stdlib/2/collections.pyi

```diff
@@ -93,10 +93,13 @@ class Counter(Dict[_T, int], Generic[_T]):
     def __iand__(self, other: Counter[_T]) -> Counter[_T]: ...
     def __ior__(self, other: Counter[_T]) -> Counter[_T]: ...
 
+_OrderedDictT = TypeVar('_OrderedDictT', bound=OrderedDict)
```

Comment:
Personally, I prefer shorter names for type variables (also this is what PEP 8 currently recommends).

Reply:
I followed the convention already established in the file by `_UserDictT`, `_UserListT`, and `_UserStringT`.  I think it makes sense to keep it consistent.

Reply:
If you really hate it as is I can do `_OdictT`.  your call.